src/c++/perf_analyzer/main.cc

```diff
@@ -278,13 +279,19 @@ Usage(char** argv, const std::string& msg = std::string())
   std::cerr
       << FormatMessage(
              " --service-kind: Describes the kind of service perf_analyzer "
-             "to generate load for. The options are \"triton\", \"tfserving\" "
-             "and \"torchserve\". Default value is \"triton\". Note in order "
-             "to use \"torchserve\" backend --input-data option must point to "
-             "a json file holding data in the following format {\"data\" : "
+             "to generate load for. The options are \"triton\", "
```

Comment:
Notice how the formatter broke this long string in strange ways. If you combine all these fragments into one long string and run the formatter again it will do a better job.